docs/deploying a sample wildfly application.md

```diff
@@ -0,0 +1,144 @@
+# Deploying a sample WildFly application
+
+User Story:
+User downloads and starts Minishift (which deploys an instance of OpenShift Origin locally) in order to launch a sample Hello World application using WildFly.
+
+<!-- MarkdownTOC -->
+
+- [Objective](#objective)
+- [Prerequisites](#prerequisites)
+- [Deploying a Hello World WildFly application](#deploying-a-hello-world-wildfly-application)
+- [Additional resources](#additional resources)
+
+
+<!-- /MarkdownTOC -->
+
+<a name="objective"></a>
+## Objective
+To deploy a Hello World application using WildFly, on an OpenShift Origin instance provisioned by Minishift.
+
+<a name="prerequisites"></a>
+## Prerequisites:
+You can create a new application in OpenShift Origin either from a source code, an image or a template. In this example we use source code along with the WildFly Imagestream to deploy a Hello World application.
```

Comment:
So if the objective is to create the application, how can then the very act of creation be a prerequisite "You can create a new application in OpenShift Origin either from a source code"? I don't get it.

 Also one cannot say "from a source code"